ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Nagios_Plugins/tasks/nagios_Ubuntu.yml

```
@@ -27,7 +27,7 @@
     state: latest
     update_cache: yes
     pkg: nagios-plugins-common
-  when: (ansible_distribution_major_version != 20)
+  when: not (ansible_distribution_major_version == "20")
```

Could we use something like this instead? Presumably the conditional will be valid for future versions >20 too which this won't catch?


```suggestion
  when: (ansible_distribution_version|version_compare('20.04', '>='))
```

---

Put in the latest commit, your syntax was a little off :-)

---

Doesn't surprise me - came off the top of my head and I trusted you to adjust it accordingly ;-) Thanks

---

:joy: You're welcome